chat-plugins/prowrestling.js

```diff
@@ -0,0 +1,195 @@
+/**
+ * Professional Wrestling Room Plugin
+ * Pokemon Showdown - http://pokemonshowdown.com/
+ *
+ * This handles submitting questions for the podcast that the room runs periodically,
+ * instead of having to use another source to ask questions.
+ *
+ * @license MIT license
+ */
+
+'use strict';
+
+const FS = require('../fs');
+const PW_DATA_PATH = 'config/chat-plugins/prowrestling-questions.json';
+
+const MAX_QUESTION_LENGTH = 500;
+
+let pwData = {};
+try {
+	let pwData = require(`..${PW_DATA_PATH}`);
```

Comment:
Using `let` here shadows the `pwData` var outside the try block's scope, preventing its value from being changed